master/cluster.go

```diff
@@ -1238,40 +1238,53 @@ func (c *Cluster) markDeleteVol(name, authKey string, force bool) (err error) {
 		return proto.ErrVolNotExists
 	}
 
-	if !c.cfg.volForceDeletion {
-		volDentryCount := uint64(0)
-		mpsCopy := vol.cloneMetaPartitionMap()
-		for _, mp := range mpsCopy {
-			// to avoid latency, fetch latest mp dentry count from metanode
-			c.doLoadMetaPartition(mp)
-			mpDentryCount := uint64(0)
-			for _, response := range mp.LoadResponse {
-				if response.DentryCount > mpDentryCount {
-					mpDentryCount = response.DentryCount
+	if isdelete {
```

Comment:
`
if !isdelete {
	serverAuthKey = vol.Owner
		if !matchKey(serverAuthKey, authKey) {
			return proto.ErrVolAuthKeyNotMatch
		}

		vol.Status = normal
		if err = c.syncUpdateVol(vol); err != nil {
			vol.Status = markDelete
			return proto.ErrPersistenceByRaft
		}
return
}
`

Reply:
Have removed the else and adjusted the code logic.